paullanza/home_away
# My events page

Added:
- My Events page
- Event cards
- Mapbox and markers

Test:
- Check the page out, create an event, try to attend some as well, see what changes on the My Events page.
- Verify code for conflicts
